lib/rsa/rsa-verify.c

```diff
@@ -319,13 +319,17 @@ static int rsa_verify_key(struct image_sign_info *info,
 	hash_len = checksum->checksum_len;
 
 #if !defined(USE_HOSTCC)
+#if CONFIG_IS_ENABLED(DM)
 	ret = uclass_get_device(UCLASS_MOD_EXP, 0, &mod_exp_dev);
```

Comment:
Looks like mod_exp_dev definition is also protected by "if !defined(USE_HOSTCC)", so won't that result in a defined but not used variable?

Reply:
Yep:
```
u-boot/lib/rsa/rsa-verify.c: In function ‘rsa_verify_key’:
u-boot/lib/rsa/rsa-verify.c:295:18: warning: unused variable ‘mod_exp_dev’ [-Wunused-variable]
  295 |  struct udevice *mod_exp_dev;
      |                  ^~~~~~~~~~~
```

Reply:
fixed